plugins/generator-1.19.4/forge-1.19.4/templates/livingentity/livingentity.java.ftl

```diff
@@ -708,6 +713,21 @@ public class ${name}Entity extends ${extendsClass} <#if data.ranged>implements R
 		}
     </#if>
 
+	<#if hasProcedure(data.boundingBoxScale) || (data.boundingBoxScale?? && data.boundingBoxScale.getFixedValue() != 1)>

```

Comment:
Usually we just check if a dynamic value (the procedure) or a static (fixed) one is defined, could you remind when this field can get a null value so that `data.boundingBoxScale??` is needed?

Reply:
Null checks in procedures are because in MCreator Procedure field is allowed to have null value since procedures were introduced so this is intentional